docs/source/api/users.rst

```diff
@@ -38,6 +38,8 @@ Request Structure
 	+-----------+----------+------------------------------------------------------------------------------------------+
 	| tenant    | no       | Return only users belonging to the :term:`Tenant` identified by tenant name              |
 	+-----------+----------+------------------------------------------------------------------------------------------+
+	| role      | no       | Return only users belonging to the :term:`Role` identified by role name                  |
+	+-----------+----------+------------------------------------------------------------------------------------------+
```

Comment:
There should be a `versionadded` directive either right above or right below this table to mention that this was unavailable in 4.0. I think our next version is ATC 4.1 - barring any urgent bugfixes.